package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "VencordDesktop",
+    "name": "vencord-desktop",
```

Comment:
this breaks everyone's settings

Reply:
oops, completely forgot that settings relied on that, let me fix that rq